### PR TITLE
Cloud FAQ improvements

### PR DESCRIPTION
**Pull request by alexeyzimarev**

### CODE REVIEW

File: docs/cloud/faq/cluster_provisioning.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Cluster Provisioning
 
-### Are there plans to support automatic VM resize? Either online or offline, but without having to go to backups.
+### Are there plans to support automatic VM resize? Either online or offline, but without having to do back up and restore the cluster.
```
Comment (mat-mcloughlin) [inline docs/cloud/faq/cluster_provisioning.md]:
```suggestion
### Are there plans to support automatic VM resize? Either online or offline, but without having to back up and restore the cluster.
```

File: docs/cloud/faq/providers.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+### Do you use locally-redundant storage (LRS) or zone-redundant storage (ZRS)?
 
-### Azure: We use AKS , to simplify setup can you expose the cluster to the public internet?
+Customer data is only stored on Premium SSD block device targets. We do not utilize Azure file or blob storage for customer data.
 
-We have no plans to expose clusters via a public IP address
+### We use AKS, to simplify setup can you expose the cluster to the public internet?
 
-### Kubernetes : Is there guidance in using Kubernetes with EvensStore cloud ? 
+We have no plans to expose clusters via a public IP address. We will be providing guidelines on how to connect AKS clusters to Event Store CLoud.
```
Comment (mat-mcloughlin) [inline docs/cloud/faq/providers.md]:
```suggestion
We have no plans to expose clusters via a public IP address. We will be providing guidelines on how to connect AKS clusters to Event Store Cloud.
```

File: docs/cloud/faq/providers.md [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 
-Guidance on  linking resources hosted in  Kubernetes on  Azure (AKS) and Google Cloud (GKE) will be published soon.
\ No newline at end of file
+## Google Cloud Platform
+
+### How can we connect GKE workloads to Event Store Cloud?
+
+Please read our [blog post](https://www.eventstore.com/blog/connecting-event-store-cloud-to-google-kubernetes-engine) on this topic. We will it to the documentation later on.
```
Comment (mat-mcloughlin) [inline docs/cloud/faq/providers.md]:
```suggestion
Please read our [blog post](https://www.eventstore.com/blog/connecting-event-store-cloud-to-google-kubernetes-engine) on this topic. We will be adding it to the documentation soon.
```

File: docs/cloud/faq/roadmap.md [hunk 4]
```diff
@@ -4,24 +4,23 @@
...
 ### What is the ETA on the tool being developed to assist in migration of existing customer data?
 
-We have a replicator that is available now. 
-Depending on your requirements this may be suitable
+We have a replication tool, which is available now. It has certain limitations, especially with performance. Get in touch, so we can help you to analyse your setup and requirements, before we can recommend using the replication tool.
 
 ### Do you have indicative performance for each of your offered cluster sizes?
 
-Yes, benchmarks will be published soon and will be available via the console.
+We are still measuring the performance of different cluster sizes and optimise the configuration for better utilisation of all the instance sizes. We will provide indicative figures on the Cloud Console as soon as we are confident in these numbers.
```
Comment (mat-mcloughlin) [inline docs/cloud/faq/roadmap.md]:
```suggestion
We are still measuring the performance of different cluster sizes and optimising the configuration for better utilisation of all the instance sizes. We will provide indicative figures on the Cloud Console as soon as we are confident in these numbers.
```

File: docs/cloud/faq/roadmap.md [hunk 5]
```diff
@@ -4,24 +4,23 @@
...
 
-Yes this on the roadmap.
+Yes, we are actively working on it.
 
-### What is the ETA on the scheduled backups functionality being implemented?
-
-This is part of the next bit of functionality we are releasing
+### What is the ETA on the scheduled backup general availability?
 
+The scheduled backup feature is in the QA phase and we are looking forward to release it soon.
```
Comment (mat-mcloughlin) [inline docs/cloud/faq/roadmap.md]:
```suggestion
The scheduled backup feature is in the QA phase and we are looking to release it soon.
```

File: docs/cloud/faq/troubleshooting.md [hunk 6]
```diff
@@ -1,14 +1,17 @@
 # Troubleshooting
 
-## No DNS Resolution of the cluster
+### The cluster DNS name does not resolve on my machine, what do I do? 
 
-Some ISP, router and DNS servers will not resolve or filter out answers to DNS queries for  `xxxx.mesdb.eventstore.cloud` because the address resolves to a private IP range.
+Some Interner providers, routers, and DNS servers will not resolve or filter out answers to DNS queries for `xxxx.mesdb.eventstore.cloud` because the DNS name resolves to a private IP range.
+
+You can check if that's the issue you're experiencing, run the following `nslookup` query. Replace the domain name by your cluster DNS name.
```
Comment (mat-mcloughlin) [inline docs/cloud/faq/troubleshooting.md]:
```suggestion
You can check if that's the issue you're experiencing by running the following `nslookup` query. Replace the domain name by your cluster DNS name.
```

File: docs/cloud/faq/sla.md [hunk 7]
```diff
@@ -1,20 +1,16 @@
...
-We plan to achieve SOC 2 and ISO 27001 by second half 2021.
+Event Store Ltd has a security policy. The security policy hasn't been independently audited yet. We plan to achieve SOC 2 and ISO 27001 by second half 2021.
 
+### Has the system been independently audited?
 
-## Has the system being independently audited?
-
-The security hasn't been independently audited yet.
-We plan to achieve SOC 2 and ISO 27001 by second half 2021.
+Our security policy hasn't been independently audited yet. We plan to achieve SOC 2 and ISO 27001 by second half 2021.
```
Comment (ylorph) [inline docs/cloud/faq/sla.md]: this is about the system itself ,  not the security policy

**Review by ylorph — APPROVED** [review]

**Pull request merged**